src/wizard/wizard-form.tsx

```diff
@@ -13,6 +13,7 @@ import useFocusVisible from '../internal/hooks/focus-visible';
 import { useEffectOnUpdate } from '../internal/hooks/use-effect-on-update';
 
 interface WizardFormProps {
+  analyticsEnabled: boolean;
```

Comment:
Since our hook is computed once and reuses cached value, we can call it as many times as we want.

Should we call it in every component instead of polluting the API with passing this prop down

Reply:
Makes sense 👍🏻